src/System.Runtime.Extensions/tests/System/AppDomainTests.cs

```diff
@@ -310,6 +305,7 @@ public void Unload()
         }
 
         [Fact]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.Uap | TargetFrameworkMonikers.UapAot, "Assembly.Load(byte[], ...) is not supported in AppX.")]
```

Comment:
this is what `git blame` has to say about that line you pointed to:
`^ef1e2ab (dotnet-bot         Fri Jan 30 14:14:42 2015) #if FEATURE_APPX`
Note it's 2015. The feature work has not been done yet. The primary purpose of going through the test failures is to find out what we need to fix. The secondary purpose is to get the tests passing. (Now in this specific case maybe there are technical reasons, but the fact that it didn't work in 2015 is not the reason)